knotx-example/knotx-example-app/config/application.conf

```diff
@@ -0,0 +1,71 @@
+modules = [
+  "server=io.knotx.server.KnotxServerVerticle"
+  "httpRepo=io.knotx.repository.HttpRepositoryConnectorVerticle"
+  "fsRepo=io.knotx.repository.FilesystemRepositoryConnectorVerticle"
+  "splitter=io.knotx.knot.splitter.FragmentSplitterVerticle"
+  "assembler=io.knotx.knot.assembler.FragmentAssemblerVerticle"
+  "hbsKnot=io.knotx.knot.templating.HandlebarsKnotVerticle"
+  "serviceKnot=io.knotx.knot.service.ServiceKnotVerticle"
+  "actionKnot=io.knotx.knot.action.ActionKnotVerticle"
+  "serviceAdapter=io.knotx.adapter.service.http.HttpServiceAdapterVerticle"
+  "actionAdapter=io.knotx.adapter.action.http.HttpActionAdapterVerticle"
+  "repoMock=io.knotx.mocks.MockRemoteRepositoryVerticle"
+  "serviceMock=io.knotx.mocks.MockServiceVerticle"
+  "gatewayKnot=io.knotx.gateway.GatewayKnotVerticle"
+  "reqProcessorKnot=io.knotx.gateway.RequestProcessorKnotVerticle"
+  "respProviderKnot=io.knotx.gateway.ResponseProviderKnotVerticle"
+]
+
+repo {
+  domain = localhost
+  port = 3001
+}
+
+service {
+  domain = localhost
+  port = 3000
+}
+
+address {
+  repo {
+    fs = knotx.core.repository.filesystem
+    http = knotx.core.repository.http
+  }
+  knot {
+    action = knotx.knot.action
+    service = knotx.knot.service
+    hbs = knotx.knot.handlebars
+    gateway = knotx.gateway.gatewayknot
+    reqProcessor = knotx.gateway.requestprocessor
+  }
+  adapter {
+    action = knotx.adapter.action.http
+    service = knotx.adapter.service.http
+  }
+}
+
+config {
+  server {
+    options.config.defaultFlow {
+      include file("./config/includes/server-default-flow.conf")
```

Comment:
Let's imagine:
```
include file (data-provider.conf)
```
then in `data-provider.conf`
```
include file (data-provider-knot.conf)
include file (data-provider-adapter-http.conf)
```
Modularity looks very promising.